test/Coinbase/Exchange/Socket/Test.hs

```diff
@@ -97,3 +110,667 @@ waitCancelThreads action1 action2 = do
         Left  a -> cancel b
         Right b -> cancel a
     return c
+
+--------------------------------------------------------------------------------
+-- OFFLINE UNIT TESTS
+--------------------------------------------------------------------------------
+-- Tests of documented WebSocket messages
+--
+-- The quasiquoted JSON objects that follow are the officially documented
+-- examples of valid messages that can be sent or received via Coinbase Pro's
+-- Websocket API as described at https://docs.pro.coinbase.com/#websocket-feed
+-- (retrieved 2019-11-5).
+--
+-- The examples below are presented in the order in which they appear in the
+-- documentation. These examples should be updated if the documentation changes.
+--
+-- NOTE: Some of the examples in the documentation are not valid JSON: they
+-- include comments, elisions, or invalid syntax such as trailing commas.
+-- They're modified here just enough to make them resemble the messages actually
+-- produced by the API.
+
+docMessageTests = testGroup "Test parsing of example messages from official\
+  \ documentation"
+  [ testCase "Parse documented \"error\" message (`ErrorMsg`)" $
+      fromJSON docErrorMsg @?= Success expectErrorMsg
+  -- , testCase "Parse documented \"subscribe\" message (`Subscribe`)" $
+  --     fromJSON docSubscribe @?= Success expectSubscribe
+  , testCase "Parse documented \"subscriptions\" message (`SubscriptionsMsg`)" $
+      fromJSON docSubscriptionsMsg @?= Success expectSubscriptionsMsg
```

Comment:
Suggestion. I think `expected` may be a better prefix than `expect` in naming these values. I think `expect` is a little confusing because it is a verb.

Reply:
Will change.